src/configure/test/suite/index.ts

```diff
@@ -0,0 +1,38 @@
+import * as glob from 'glob';
+import * as Mocha from 'mocha';
+import * as path from 'path';
+
+export function run(): Promise<void> {
+	// Create the mocha test
+	const mocha = new Mocha({
+		ui: 'bdd',
+		color: true
+	});
+
+	const testsRoot = path.resolve(__dirname, '..');
+
+	return new Promise((c, e) => {
+		glob('**/**.test.js', { cwd: testsRoot }, (err, files) => {
```

Comment:
we should only glob for files under `suite` folder

Reply:
Okay, will change it.